src/types/Todo.ts

```diff
@@ -1,4 +1,5 @@
 export interface Todo {
+  [x: string]: any;
```

Comment:
Same about any

Reply:
that code is not mine, it was already here

Reply:
A green line means that this line has been added in this PR. And in any case, it is now your project and you are responsible for it.